docs/src/overview.rst

```diff
@@ -20,6 +20,7 @@ This guide contains the following chapters:
   CriticalUp.
 * *Using CriticalUp* describes how to use CriticalUp to manage Ferrocene
   installations.
+* *Ferrocene Subscription* describes how to manage your subscription.
```

Comment:
```suggestion
* *Managing Subscriptions* describes how to manage your subscription.
```

Seems the page name is this, not "Ferrocene Subscriptions"

Reply:
Thanks! These manual mini TOCs are a pet hate of mine, for precisely this reason.